src/esockd_connection_sup.erl

```diff
@@ -265,10 +265,19 @@ connection_crashed(_Pid, Reason, _State) when is_atom(Reason) ->
     count_shutdown(Reason);
 connection_crashed(_Pid, {shutdown, Reason}, _State) when is_atom(Reason) ->
     count_shutdown(Reason);
+connection_crashed(Pid, {shutdown, {ssl_error, Reason}}, State) ->
+    count_shutdown(ssl_error),
+    log(info, ssl_error, Reason, Pid, State);
+connection_crashed(Pid, {shutdown, #{shutdown_count := Key} = Reason}, State) when is_atom(Key) ->
```

Comment:
Nit: maybe `shutdown_reason` would fit better than `shutdown_count`, as `Key` is not really a counter?

Reply:
this makes the code easier to trace.
grep shutdown_count will lead whoever wants to quickly find out where it is used to this module.